source/extensions/watchdog/abort_action/abort_action.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <chrono>
+
+#include "envoy/extensions/watchdog/abort_action/v3alpha/abort_action.pb.h"
+#include "envoy/server/guarddog_config.h"
+#include "envoy/thread/thread.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Watchdog {
+namespace AbortAction {
+
+/**
+ * A GuardDogAction that will terminate the process by sending SIGABRT to the
```

Comment:
Perhaps some/all of this comment block should be included with the proto config, so it gets included in generated docs.

Reply:
Done. Added additional information from the action's c++ implementation as you suggested.